types/bitcoin.types.ts

```diff
@@ -107,3 +110,40 @@ export const callOptionsSchema = z
     message: typesAndDataExclusivityRefineRule.message,
     path: typesAndDataExclusivityRefineRule.path,
   });
+
+export const memoDepositAndCallOptionsSchema = z.object({
+  amount: validAmountSchema,
+  bitcoinApi: z.string().url(),
+  data: hexStringSchema.optional(),
+  gasPriceApi: z.string().url(),
+  gateway: z.string(),
+  name: z.string().optional().default(DEFAULT_ACCOUNT_NAME),
+  networkFee: z.string(),
+  privateKey: z.string().optional(),
+  receiver: z.string().optional(),
+});
+
+export const memoDepositOptionsSchema = z.object({
+  amount: z.string().refine((val) => !isNaN(Number(val)) && Number(val) > 0, {
+    message: "Amount must be a valid positive number",
+  }),
+  bitcoinApi: z.string().url(),
+  data: hexStringSchema.optional(),
+  gasPriceApi: z.string().url(),
+  gateway: z.string(),
+  name: z.string().optional().default(DEFAULT_ACCOUNT_NAME),
+  networkFee: z.string(),
+  privateKey: z.string().optional(),
+  receiver: z.string().optional(),
+});
+
+export const memoCallOptionsSchema = z.object({
+  bitcoinApi: z.string().url(),
+  data: hexStringSchema.optional(),
+  gasPriceApi: z.string().url(),
+  gateway: z.string(),
+  name: z.string().optional().default(DEFAULT_ACCOUNT_NAME),
+  networkFee: z.string(),
+  privateKey: z.string().optional(),
+  receiver: z.string().optional(),
```

Comment:
Is this always an evm compatible address? If that's the case we can reuse the `evmAddressSchema`.

Reply:
Good call! 3d5b7c7